thanosql/_base_client.py

```diff
@@ -125,14 +125,30 @@ def _request(
                     message = response_json["error"].get("message", message)
 
                 if code in {400, 405, 422}:
-                    raise thanosql_error.ThanoSQLValueError(message=message)
+                    raise thanosql_error.ThanoSQLValueError(
+                        message=f"Invalid input: {message}"
```

Comment:
I'm not sure if the short intro is needed, but it was inspired by the recent 413 addition -> even if caught, user still won't know that the cause of the error is a big file, as the message is just "... please contact ThanoSQL team".

As seen in some examples in the description, the intro can make things clearer in some cases, and sound redundant in the rest.